ui/app/pages/swaps/select-quote-popover/index.scss

```diff
@@ -39,54 +39,43 @@
 
   &__sort-list {
     display: flex;
-    flex-flow: column;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  &__column-headers,
+  &__row {
+    display: flex;
     align-items: center;
+    padding-left: 20px;
+    width: 100%;
+
+    > * + * {
+      margin-left: 8px;
```

Comment:
This was the fix for the core issue. There is now a minimum of 8px between each column.

I tried with 6px first but the numbers still looked too close together - it didn't seem legible enough.

Reply:
just to make sure i understand... this is targeting any element that is a direct descendant of the parent that is proceeded by any element? I guess the cells could be any element? would it be better to target a specific class or set a margin-left: 0 on the :first-child?

Reply:
Perhaps :thinking: . I had been assuming that the direct descendent of any row is a grid cell - seemed like a decent assumption. There is no "grid cell" class I could refer to at the moment either.

I think using `:first-child` as you described would achieve the same result with more lines? I had done it this way because I was copying @rekmarks to be honest :sweat_smile: It seemed concise.